prisma/schema.prisma

```diff
@@ -4,6 +4,10 @@
 // Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
 // Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
 
+generator prismaClassGenerator {
+  provider = "prisma-class-generator"
+}
```

Comment:
There should be an ouptut

generator prisma_class_generator {
  provider      = "prisma-class-generator"
  output        = "./../src/common/prisma/gen" // You can update the location as you want
}